code/stable/glassbr/src/csharp/Calculations.cs

```diff
@@ -3,60 +3,58 @@
 using System.Collections;
 using System.Collections.Generic;
 
-namespace GlassBR {
```

Comment:
Any reason the `namespace` disappeared?

Reply:
I saw that the code worked without it and didn't see why it was necessary/useful, so I dropped it. If there's a reason for it, I can look into adding it back, though I think this is something that will be much easier to implement once we have `Reader`s for our instances, so that we can read the `Package` name whenever we need it.